benchmark/compiler/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(compiler_benchmark basic.cc)
```

Comment:
Also, we tend to fully qualify target names, as this library if for downstream consumption and we might get some name collisions at some point? For example `sourcemeta_jsontoolkit_jsonschema_benchmark_xxxxxx`?

Reply:
We could eventually have nice macros at Noa to hide all of these things